api/v1beta1/hyperconverged_types.go

```diff
@@ -757,6 +760,12 @@ type ApplicationAwareConfigurations struct {
 	AllowApplicationAwareClusterResourceQuota bool `json:"allowApplicationAwareClusterResourceQuota,omitempty"`
 }
 
+type HigherWorkloadDensityConfiguration struct {
+	// MemoryOvercommitRatio defines the ratio of memory overcommit for the vitrual machine workloads
+	// The ratio can be used in order to increase virtual machine density
+	MemoryOvercommitRatio *string `json:"memoryOvercommitRatio,omitempty"`
```

Comment:
Why using string here? we only support float value anyway, so we won't need to convert later, and the API server will make sure it's a floating number value.
```suggestion
	MemoryOvercommitRatio *float64 `json:"memoryOvercommitRatio,omitempty"`
```

Reply:
@nunnatsa Hi! I changed to `float64` and got the following error when running `make sanity`:
```
panic: /home/ibezukh/go/src/kubevirt.io/hyperconverged-cluster-operator/api/v1beta1/hyperconverged_types.go:768:24: found float, the usage of which is highly discouraged, as support for them varies across languages. Please consider serializing your float as string instead. If you are really sure you want to use them, re-run with crd:allowDangerousTypes=true
```

Reply:
OK. thanks.